### PR TITLE
Add support for firing metrics via the new metrics posting route on the metrics API.

### PR DESCRIPTION
**Pull request by hodgestar**
The Vumi Go metrics API gained support for firing metrics in https://github.com/praekelt/go-metrics-api/pull/16. We need to add support for calling that to the metrics client.

### CODE REVIEW

**Comment by hodgestar** [comment]
Ready for review but we should probably decide how https://github.com/praekelt/go-metrics-api/issues/18  will be resolved before landing this.

**Comment by hodgestar** [comment]
Ready for review.

File: go_http/metrics.py [hunk 1]
```diff
@@ -74,3 +74,39 @@ def get_metric(self, metric, start, interval, nulls):
...
         return self._api_request("GET", "metrics/", payload)
+
+    def fire(self, metrics):
+        """
+        Fire metrics.
+
+        :param dict metrics:
+            A mapping of metric names to floating point metric values.
+
+        When metrics are fired they may specify an aggregator. The
```
Comment (rudigiesler) [inline go_http/metrics.py]: s/may/must

**Comment by rudigiesler** [comment]
Just the one comment from me

**Comment by hodgestar** [comment]
Ready for re-review.

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**